homework/content.js

```diff
@@ -1,37 +1,78 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
-
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+(function(exports) {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
+  /**
+  * Creates an instance of ContentManager.
+  *
+  * @constructor
+  * @this {ContentManager}
+  */
+  var ContentManager = function () {
+    this._wrapper = document.querySelector('#note-content-wrapper');
   }
 
-  function drawNote(note) {
-    var title = note.title;
-    var h = document.createElement('h2');
-    h.textContent = title;
-    var passages = note.passages;
-    var buff = document.createDocumentFragment();
-    passages.forEach(function(passage) {
-      var p = document.createElement('p');
-      p.classList.add('note-passage');
-      p.textContent = passage;
-      buff.appendChild(p);
-    });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
+  ContentManager.prototype = {
+
+    /**
+    * Bind 'note-open' event to initialize the ContentManager.
+    *
+    * @this {ContentManager}
+    */
+    start() {
+      window.addEventListener('note-open', (function(event) {
```

Comment:
You need to bind the listener with the `this`, rather than an anonymous function like this.